src/battle/mod.rs

```diff
@@ -0,0 +1,70 @@
+use bevy::prelude::*;
+
+use crate::AppState;
+use crate::APPSTATES;
+
+/// Marker for despawning when exiting `AppState::Battle`
+pub struct StateCleanup;
+
+pub struct BattlePlugin;
+impl Plugin for BattlePlugin {
+    fn build(&self, app: &mut AppBuilder) {
+        app.on_state_enter(APPSTATES, AppState::Battle, setup_battle.system())
+            // NOTE: Uses same camera as overworld
```

Comment:
```suggestion
            // TODO: Uses same camera as overworld
```

`TODO` gets picked by most IDEs worth their salt. `NOTE` is rarer.